.vscodeignore

```diff
@@ -84,3 +86,4 @@
 
 # resources
 !resources/contentCreator/createDevfile/devfile-template.txt
+!resources/contentCreator/createDevcontainer/devcontainer-template.txt
```

Comment:
This path should be changed to `!resources/contentCreator/createDevcontainer/.devcontainer/devcontainer-template.txt` and paths for the docker and podman templates also should be added. 
`!resources/contentCreator/createDevcontainer/.devcontainer/podman/devcontainer-template.txt`
`!resources/contentCreator/createDevcontainer/.devcontainer/docker/devcontainer-template.txt`